perfect_roll.py

```diff
@@ -0,0 +1,20 @@
+def perfect_roll():
+    """
+    function that generates perfect numbers infinitely
+    :return: generator of perfect number
+    """
+    i = 6
```

Comment:
1. Why do you start from 6?
(If its because there arent any perfect numbers below 6 you should write it in a side comment).
2. What is i? (give it a better name).